### PR TITLE
Added Ability to delete all in Notification Center

### LINKED ISSUE #8867: [UI] Notification Center: Ability to delete all
### Current Behavior
173 events - too many to mark read individually. The prior version of the Notification Center has a "clear all" action.

### Desired Behavior
1. Offer the ability to delete all events.

### Acceptance Tests
1. Single-click to delete all.

### Mockups
1. @Modern-artist is there such an action in the mockups?

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by pushkarm029**
**Notes for Reviewers**

This PR fixes #8867 

Added a Button to delete all events in one click.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Yashsharma1911 — COMMENTED** [review]
@Pushkarm029 thankyou for taking this up, can you share video or screenshot of your changes

File: ui/components/NotificationCenter/index.js [hunk 1]
```diff
@@ -361,28 +368,47 @@ const BulkActions = () => {
...
+          <Box
+            style={{
+              display: 'flex',
+              gap: '1rem',
+              alignItems: 'center',
+              flexDirection: 'row-reverse',
+            }}
+          >
+            <BulkActionButton
+              tooltip="Delete All notifications"
```
Comment (Yashsharma1911) [inline ui/components/NotificationCenter/index.js:382]:
```suggestion
              tooltip="Delete all notifications"
```

Comment (Yashsharma1911) [reply ui/components/NotificationCenter/index.js:382]: A small change

**Comment by Pushkarm029** [comment]
> @Pushkarm029 thankyou for taking this up, can you share video or screenshot of your changes


https://github.com/meshery/meshery/assets/81290616/fb88c68c-f741-44c1-baf7-be0706cf2813

**Comment by aabidsofi19** [comment]
@Yashsharma1911  @Pushkarm029  this needs to be implemented on backend , rightnow it is only doing a select all for the current loaded notification  which we can do by  doing a select all and delete .  we need to enhance the current bulk op endpoint for delete_all , change_status for all , etc ops .  // @MUzairS15

**Comment by Pushkarm029** [comment]
@aabidsofi19 Working on it

**Comment by saurabh100ni** [comment]
Thanks for raising PR @Pushkarm029, please add this in [Docs](https://github.com/meshery/meshery/pull/9716/files) as your item to discuss it in the Meshery Development Meet today at 8:30 pm IST.
Check out [Community Calendar ](https://layer5.io/community/calendar)for meet Details.

**Comment by leecalcote** [comment]
It would be great to have this demo'ed at tomorrow's project meeting.

**Comment by leecalcote** [comment]
@Pushkarm029, should we count you in today?

**Comment by Pushkarm029** [comment]
@leecalcote Yeah, please do

**Comment by leecalcote** [comment]
@Pushkarm029 ok, drop your agenda item into the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit).

**Comment by leecalcote** [comment]
> @Yashsharma1911 @Pushkarm029 this needs to be implemented on backend , rightnow it is only doing a select all for the current loaded notification which we can do by doing a select all and delete . we need to enhance the current bulk op endpoint for delete_all , change_status for all , etc ops . // @MUzairS15

@Yashsharma1911 or @MUzairS15 doing this?

**Comment by leecalcote** [comment]
@Yashsharma1911 might be...

**Comment by leecalcote** [comment]
@Pushkarm029 what do we have left to tie-off here?

**Comment by Pushkarm029** [comment]
If we want this functionality implemented in the frontend, it's good to go. Else., We have to implement this in the backend.
I see it might be an urgent issue, so we can use this frontend implementation until we don't have it implemented in the backend.

**Pull request closed without merging**